index.html

```diff
@@ -0,0 +1,310 @@
+                                <li class="skill-card-tech-tag selenium"><p class="skill-card-tech-tag-text">Selenium</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/xyyakzgb" method="POST" id="form-content">
+                <div class="form-row" id="first-form-row">
+                    <input type="text" name="FirstName" max-length="30" name="first-name" placeholder="First Name" class="form-row-column" required>
+                    <input type="text" name="SecondName" max-length="30" name="last-name" placeholder="Last Name" class="form-row-column" required>
+                </div>
+                <input type="text" name="FullName" max-length="30" placeholder="Full Name" class="form-row" id="ignoreindesktop" required>
+                <input type="email" name="email" max-length="30" placeholder="Email Address" id="" class="form-row" required>
+                <textarea name="comment" id="" cols="30" rows="10" max-length="500" placeholder="Typing..." class="form-row textareas"></textarea>
```

Comment:
- [ ] This form can be submitted with a black textarea field. Please add required  to it. This is a project requirement. 👏🏾 
![ins](https://user-images.githubusercontent.com/99656687/220355616-79303735-5854-4e07-a80d-1c3216f12663.png)

Reply:
- [ ] You have done great implementing your project this far, you just have a little padding issues to fix. Please give your form input fields some padding to match the design.

**Design**
![des](https://user-images.githubusercontent.com/99656687/220356729-27723353-c9ce-4e1c-97d9-72293c901580.png)
**Your implementation**
![des2](https://user-images.githubusercontent.com/99656687/220357014-ceea633a-848d-46b3-ba4a-cea5c217eeba.png)